### PR TITLE
Image Component

### PR DESCRIPTION
**Pull request by shivampandey0**
https://user-images.githubusercontent.com/51923995/152095768-38ad2a8a-b569-47b5-8bb1-fad0aaa06640.mp4

- Image Responsive
- Image Rounded

### CODE REVIEW

File: css/elements/image.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.img-responsive{
+    width: 100%;
+    height: auto;
+}
+
+.img-rounded{
+    border-radius: 50%;
```
Comment (lannisterrr) [inline css/elements/image.css]: in our component library we using this a lot of the times , you can make it a separate variable.

Comment (shivampandey0) [reply css/elements/image.css]: Thanks will do that

File: css/index.html [hunk 2]
```diff
@@ -14,8 +14,46 @@
...
+                    <img class="img-responsive" src="./assets/nike.jpeg" alt="Nike">
+                </div>
+                <div>
+                    <img class="img-responsive" src="./assets/nike.jpeg" alt="Nike">
+                </div>
+                <div>
+                    <img class="img-responsive" src="./assets/nike.jpeg" alt="Nike">
+                </div>
+            </div>
+            <div class="flex-row is-center gap-1">
```
Comment (arp99) [inline css/index.html]: What does this class **is-center** do?

File: css/layout/flexbox.css [hunk 3]
```diff
@@ -8,10 +8,15 @@
   flex-direction: column;
 }
 
-.flex-wrap {
+.is-wrap {
   flex-wrap: wrap;
 }
```
Comment (arp99) [inline css/layout/flexbox.css]: flex-wrap sounds more meaningful than is-wrap

**Review by Hamdrive — COMMENTED** [review]
Code looks clean and good, other reviewers have already given valid suggestions
Would also suggest to create a rounded corner image as well

File: css/layout/flexbox.css [hunk 4]
```diff
@@ -8,10 +8,15 @@
...
 
+.is-center{
+  justify-content: center;
+  align-items: center;
+}
+
```
Comment (arp99) [inline css/layout/flexbox.css]: Similary here class name **is-center** seems like a question, can use better names

**Review by arp99 — COMMENTED** [review]
Add some sanity checks and suggestions

File: css/index.html (hunk 2 above)
Comment (shivampandey0) [reply css/index.html]: It's a layout class in flexbox.css needed to center children of flexbox.
It was needed for the container and is not a part of images only.

File: css/layout/flexbox.css (hunk 3 above)
Comment (shivampandey0) [reply css/layout/flexbox.css]: Did this to shorten the name. can make it flex-wrap if it conflicts with some other class in the future.

File: css/layout/flexbox.css (hunk 4 above)
Comment (shivampandey0) [reply css/layout/flexbox.css]: okay, will change names to be self-explanatory.

File: css/index.html (hunk 2 above)
Comment (arp99) [reply css/index.html]: Yeah is-center sounds like a question, the naming convention could have been better

File: css/layout/flexbox.css (hunk 3 above)
Comment (arp99) [reply css/layout/flexbox.css]: is-wrap sounds like a question, so flex-wrap sounds more meaningful

**Pull request merged**